### PR TITLE
set up get customers controller/helper sandbox

### PR DESCRIPTION
**Pull request by wilsjame**

### CODE REVIEW

**Review by Ipswich — COMMENTED** [review]
I dig it. There look to be a lot of references to SearchSpring_Feed sprinkled around, I kind of assume it doesn't cause any issues, but we should probably scrub/update those sooner than later.

**Comment by wilsjame** [comment]
> I dig it. There look to be a lot of references to SearchSpring_Feed sprinkled around, I kind of assume it doesn't cause any issues, but we should probably scrub/update those sooner than later.

Good call, best to keep it clean early. I'll start doing some renaming 👍

File: Helper/Customer.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        $this->request = $request;
+        $this->response = $response;
+
+        $this->customerFactory = $customerFactory;
+        $this->customer = $customers;
+
+        $this->dateRange = $this->request->getParam('dateRange', 'All');
+    }
+
+    // sanbox >>>>>>
```
Comment (Ipswich) [inline Helper/Customer.php:45]: Left over merge comment or something

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "description": "Module to generate a SearchSpring CSV Feed",
+    "type": "magento2-module",
+    "version": "1.1.4",
+    "license": "GPL-3.0",
+    "authors": [
+        {
+            "email": "james@searchspring.com",
+            "name": "James Bathgate"
+        }
+    ],
```
Comment (Ipswich) [inline composer.json:12]: Could probably update this at some point

**Review by Ipswich — DISMISSED** [review]
Looks like really good progress, I'll pull and start fiddlin'. Good job on updating the readme as you go as well, that'll be helpful in the future I'm sure.

File: Helper/Customer.php (hunk 1 above)
Comment (wilsjame) [reply Helper/Customer.php:45]: Oh snap haha, that was my comment. Didn't realize it was the same format as merge comments.

**Review by Ipswich — APPROVED** [review]

**Pull request merged**